src/functions/isUserId.ts

```diff
@@ -0,0 +1,21 @@
+import { Request, Response, NextFunction } from 'express';
+import { db } from '../index';
+import Collections from '../constants/collections';
+import handleError from '../utils/handleError';
+
+export const isUserId = async(req: Request, res: Response, next: NextFunction): Promise<Response> => {
```

Comment:
Is it utils using for check existence of user?

Reply:
no, it's basically checking user id